packages/cosmic-swingset/src/chain-main.js

```diff
@@ -90,6 +90,19 @@ const makeBootMsg = initAction => {
   };
 };
 
+/**
+ * Extract local Swingset-specific configuration which is
+ * not part of the consensus.
+ *
+ * @param {any} resolvedConfig
```

Comment:
We probably should type this somewhere kept in sync with the golang side `SwingsetConfig`. Arguably #9987 should have done that, but I didn't think of it.

```js
/**
 * The swingset config object parsed and resolved by cosmos in
 * `golang/cosmos/x/swingset/config.go`. The shape should be kept in sync
 * with `SwingsetConfig` defined there.
 * 
 * @typedef {object} CosmosSwingsetConfig
 * @property {string} [slogfile]
 * @property {number} [maxVatsOnline]
 */
```

Then here you can do 
```suggestion
 * @param {CosmosSwingsetConfig} [resolvedConfig]
```